Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:8 as builder
+
+WORKDIR /root/
```

Comment:
style: I'm personally a fan of `/app` since `/root` can sometimes have stuff in it already from base image (I have seen `/root` map to `~`).

Reply:
Agreed, I'll update.